kernel/sched/rr.c

```diff
@@ -7,8 +7,10 @@
 static LIST_HEAD(rr_runq);
 extern struct thread_info *thread_idle;
 
-int sched_rr_init(void)
+int sched_rr_init(struct thread_info *thread)
 {
+    if (thread)
```

Comment:
Can we initialize without checking `thread`? That is, we can explicitly arrange the order.

Reply:
Why not check about `thread` is NULL or not (or maybe checking the type is `struct thread_info`), is there any benefit to put a NULL into runqueue?